src/lib_proj.cmake

```diff
@@ -68,7 +68,7 @@ set(SRC_LIBPROJ_PROJECTIONS
   projections/gall.cpp
   projections/labrd.cpp
   projections/lsat.cpp
-  projections/misrsom.cpp
+  projections/genericsom.cpp
```

Comment:
we should probably just drop the "generic" in the file name and projection method id. Generic is kind of implied

Reply:
Will do.

Reply:
Changed name and methods to remove generic prefix.